### PR TITLE
Fixing 89 typos

### PR DESCRIPTION
**Pull request by ml-evs**
### Summary

Fixes a lot of typos, spelling errors and capitalisation in any markdown files under the `./book/` directory.

I spotted a typo in a section heading, so thought I'd use `aspell` to check for any others. This awakened a burning desire within me to fix 88 other typos throughout the `./book/` folder. I've provided the `.words.lst` file generated by `aspell` that ignores many acronyms and names such that there are no further errors reported when running the following command from the root of the repository.

```bash
for f in $(find -name "*.md"); do
    aspell check -x -p ./book/.words.lst -l en_GB -M -m $f
done
```

I tried to avoid changing any valid non-British English spellings. The most picky thing I changed was capitalisation of e.g. Ubuntu/Jupyter instead of ubuntu/jupyter, and BitBucket/GitHub/GitLab instead of Bitbucket/Github/Gitlab.

### List of changes proposed in this PR (pull-request)

* [x] Fix multiple "obvious" spelling errors and typos
* [x] Fixed a couple of grammar errors
* [x] Fixed formatting of some branded names/proper nouns

### What should a reviewer concentrate their feedback on?

- [x] Firstly, sorry. Hope this isn't too much of a pain to review...
- [ ] Whether you want the `.words.lst` file included in this repository, or whether you want to just cherry-pick 60833fb as a standalone set of fixes.
- [ ] Interactions with other open PRs that fix any spelling/grammar might be a pain (e.g. I notice that #1466 runs some of the same files through grammarly and makes many/all the changes from this PR, plus further improvements).
- [x] Whether some changes are inappropriate, e.g. typo fixes in the contribution profiles written by specific people.

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by ml-evs** [comment]
aaaand the CI already fails. Looks like fixing "ie" to "i.e." triggered the bad Latin phrase checker.

**Review by malvikasharan — APPROVED** [review]
All of these changes are fantastic. Thank you @ml-evs. I will fix the Latin error, and then happy to merge this.

**Comment by malvikasharan** [comment]
As for the wordlist, I don't think we will continue to maintain that.

I think @sgibson91 had in the past opened this issue: https://github.com/alan-turing-institute/the-turing-way/issues/1229, which we decided to not add in the CI. We think that they will actually not help authors by failing wrongly as they all write in different English. In the end, it's really up to the reviewers to review the content and help fix spelling errors - or a contributor like you, who can contribute to help us improve :)

**Comment by malvikasharan** [comment]
@all-contributors please add @ml-evs for bug and typo fix.

File: book/website/reproducible-research/open.md [hunk 1]
```diff
@@ -49,12 +49,7 @@ There are five main schools of thought motivating open practices to benefit rese
 | Public | Research needs to be made accessible to the public. | Making research accessible for citizens. |
 
 Open practices also benefit the researchers that propagate them. 
-For example, there is evidence [(Mckiernan et al. 2016)](https://elifesciences.org/articles/16800) that open access articles are cited more often, as shown by the metastudy presented in the figure below.
-
-
-| ![open_access_citations](../figures/open_access_citations.jpg) |
-| -----------------------------------------------------|
-| The relative citation rate (OA: non-OA) in 19 fields of research. This rate is defined as the mean citation rate of OA articles divided by the mean citation rate of non-OA articles. Multiple points for the same discipline indicate different estimates from the same study or estimates from several studies. (See footnote 1 for references.) |
+For example, there is evidence {cite}`Mckiernan et al. 2016<McKiernan2016Open>` that open access articles are cited more often, as shown by the metastudy presented in the figure below.
```
Comment (ml-evs) [inline book/website/reproducible-research/open.md:52]:
```suggestion
For example, there is evidence {cite}`McKiernan et al. 2016<McKiernan2016Open>` that open access articles are cited more often, as shown by the metastudy presented in the figure below.
```

Thanks for spotting this (and adding the reference), this is the one name that I noticed as being incorrectly formatted.

File: book/README.md [hunk 2]
```diff
@@ -11,13 +11,13 @@ Everything else is in the `website/` directory.
 
 - The table of contents (TOC) defines the order of chapters as they appear in the book.
 To change the TOC, please edit the `website/_toc.yml` file with correct information on filenames and their relative locations in this repository.
-Documentation on controlling the TOC structure can be found on the [jupyter book website](https://jupyterbook.org/customize/toc.html).
+Documentation on controlling the TOC structure can be found on the [Jupyter book website](https://jupyterbook.org/customize/toc.html).
 - Same applies for more general configuration using `website/_config.yml`.
-Documentation on configuring book settings can be found on the [jupyter book website](https://jupyterbook.org/customize/config.html).
+Documentation on configuring book settings can be found on the [Jupyter book website](https://jupyterbook.org/customize/config.html).
```
Comment (ml-evs) [inline book/README.md:16]:
```suggestion
Documentation on configuring book settings can be found on the [Jupyter Book website](https://jupyterbook.org/customize/config.html).
```

File: book/README.md [hunk 3]
```diff
@@ -11,13 +11,13 @@ Everything else is in the `website/` directory.
 
 - The table of contents (TOC) defines the order of chapters as they appear in the book.
 To change the TOC, please edit the `website/_toc.yml` file with correct information on filenames and their relative locations in this repository.
-Documentation on controlling the TOC structure can be found on the [jupyter book website](https://jupyterbook.org/customize/toc.html).
+Documentation on controlling the TOC structure can be found on the [Jupyter book website](https://jupyterbook.org/customize/toc.html).
```
Comment (ml-evs) [inline book/README.md:14]:
```suggestion
Documentation on controlling the TOC structure can be found on the [Jupyter Book website](https://jupyterbook.org/customize/toc.html).
```

**Review by ml-evs — COMMENTED** [review]
Just noticed a couple more tweaks that I will commit myself via these suggestions

File: book/website/reproducible-research/open.md (hunk 1 above)
Comment (malvikasharan) [reply book/website/reproducible-research/open.md]: Ah thank you!

**Comment by malvikasharan** [comment]
Let me know when you are ready for me to merge this. :)

**Comment by ml-evs** [comment]
> Let me know when you are ready for me to merge this. :)

I think it's good from my end now; just reran aspell and found that it doesn't like comments in the `.words.lst`, so I updated the comment to reflect that. Also found one other typo that had gone undetected as it clashed with 'th' being added to the word list as a result of the HTML tag `<th>` (not a perfect system!).

Thanks for the careful review of this!

**Pull request merged**